src/main/scala/io/iohk/ethereum/network/NodeInfo.scala

```diff
@@ -0,0 +1,11 @@
+package io.iohk.ethereum.network
+
+import java.net.InetSocketAddress
+
+import org.spongycastle.crypto.AsymmetricCipherKeyPair
+import org.spongycastle.crypto.params.ECPublicKeyParameters
+
+// TODO: what if listen port changes? or server is stopped?
```

Comment:
yes, but I'm still thinking about good approach for that. It's basically a config that can change it's value over time of running app, any suggestions?